src/main/java/uk/gov/hmcts/cmc/claimstore/events/EventProducer.java

```diff
@@ -88,4 +89,8 @@ public void createRedeterminationEvent(Claim claim, String authorisation, String
     public void createInterlocutoryJudgmentEvent(Claim claim) {
         publisher.publishEvent(new InterlocutoryJudgmentEvent(claim));
     }
+
+    public void createRejectOrganisationPaymentPlanEvent(Claim claim) {
```

Comment:
@rattur do we need this logic now? I dont think we do

Reply:
We can remove it yes